specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,268 @@
+
+Both methods clear the data for the associated profile in which the method is called on.
+
+# Examples
+
+## Win32 C++
+```cpp
+
+/// Function to clear the autofill data from the last hour
+void ClearAutofillData() 
+{
+    wwil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
+
+    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
+    if (webview7)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webview7->get_Profile(&profile));
+        double endTime = (double)std::time(nullptr);
+        double startTime = endTime - 3600;
+        /// Get the current time and offset the current time by 3600 seconds to clear the data 
+        /// from the start time (one hour ago), until the end time (present time).
+        /// This will clear the data for the last hour.
+        COREWEBVIEW2_BROWSING_DATA_KINDS dataKinds = (COREWEBVIEW2_BROWSING_DATA_KINDS)
+            (COREWEBVIEW2_BROWSING_DATA_KINDS_GENERAL_AUTOFILL |
+            COREWEBVIEW2_BROWSING_DATA_KINDS_PASSWORD_AUTOSAVE));
+        CHECK_FAILURE(profile->ClearBrowsingDataInTimeRange(
+            dataKinds, startTime, endTime,
+            Callback<ICoreWebView2StagingClearBrowsingDataCompletedHandler>(
+                [this](HRESULT error)
+                    -> HRESULT {
+                    LPCWSTR result = error == S_OK ? L"Succeeded" : L"Failed";
+                    RunAsync([this, result]() {
+                        MessageBox(nullptr, result, L"Clear Browsing Data", MB_OK);
+                    });
+                    return S_OK;
+                })
+                .Get()));
+    }
+}
+```
+
+### .NET, WinRT
+```c#
+
+private void ClearAutofillData() 
+{
+    CoreWebView2Profile profile;
+    if (webView.CoreWebView2 != null) 
+    {
+        profile = webView.CoreWebView2.Profile;
+        double endTime = DateTimeOffset.Now.ToUnixTimeSeconds();
+        // Get the current time which is the time in which the browsing data will be cleared until.
+        double startTime = endTime - 3600;
+        // Offset the current time by 3600 seconds to clear the browsing data from the last hour.
+        CoreWebView2BrowsingDataKinds dataKinds = (CoreWebView2BrowsingDataKinds)(CoreWebView2BrowsingDataKinds.GeneralAutofill | CoreWebView2BrowsingDataKinds.PasswordAutosave);
+        await profile.ClearBrowsingDataAsync(dataKinds, startTime, endTime);
+    }
+}
+
+```
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+
+# API Details
+
+## Win32 C++
+
+```IDL
+interface ICoreWebView2ClearBrowsingDataCompletedHandler;
+
+/// Specifies the datatype for the
+/// `ICoreWebView2Profile::ClearBrowsingData` method.
+[v1_enum]
```

Comment:
Does it help to say `[flags]` here?

Reply:
As far as I know the COM IDL doesn't have [flags] attribute - at least its not in the IDL documentation on msdn?